url/urlsearchparams-constructor.html

```diff
@@ -24,7 +24,7 @@
 
 test(() => {
     params = new URLSearchParams(DOMException.prototype);
-    assert_equals(params.toString(), "Error=")
+    assert_equals(params.toString(), "INDEX_SIZE_ERR=1&HIERARCHY_REQUEST_ERR=3&WRONG_DOCUMENT_ERR=4&INVALID_CHARACTER_ERR=5&NO_MODIFICATION_ALLOWED_ERR=7&NOT_FOUND_ERR=8&NOT_SUPPORTED_ERR=9&INUSE_ATTRIBUTE_ERR=10&INVALID_STATE_ERR=11&SYNTAX_ERR=12&INVALID_MODIFICATION_ERR=13&NAMESPACE_ERR=14&SECURITY_ERR=18&NETWORK_ERR=19&ABORT_ERR=20&URL_MISMATCH_ERR=21&QUOTA_EXCEEDED_ERR=22&TIMEOUT_ERR=23&INVALID_NODE_TYPE_ERR=24&DATA_CLONE_ERR=25")
```

Comment:
So it looks like we have the following extras:
DOMSTRING_SIZE_ERR=2
NO_DATA_ALLOWED_ERR=6
INVALID_ACCESS_ERR=15
VALIDATION_ERR=16
TYPE_MISMATCH_ERR=17